docs/manage/_content.rst

```diff
@@ -93,16 +95,19 @@ If the computer where Kolibri is running has an Internet connection with the suf
     .. figure:: img/import-CC.png
   	  :alt: Content import progress bar will display the percentage of the download, and the Close button once it's finished 
 
-#. If you need to import content from a **private/unlisted** channel, click on **Try adding the token** link above the channel list.  
-#. Enter the **channel token/ID** received from the channel curator on Kolibri Studio.
+.. _import_with_token:
```

Comment:
@ivanistheone I squeezed in a `manage.html#import-with-token` shortcut in the meantime, until I finish the refactor from #45 :wink: